boardswarm-cli/src/ui_term.rs

```diff
@@ -5,13 +5,31 @@ use ratatui::{
     widgets::Widget,
 };
 
+#[derive(Clone, Copy, Debug)]
+pub enum TerminalState {
```

Comment:
These are not states; just device modes and device modes are not enums they're aribitrary strings (on/off are just conventions); Reset is not a state either and neither is initial.

Reply:
OTOH if I want to display different modes with different attributes (e.g. on would be black on green, while off would be black on red), then I need to start interpreting the string, which seems not the greatest option. What's more, future localization might mean that actual strings passed are unknown to the programmer.